src/pages/instances/actions/CreateImageFromInstanceBtn.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { ActionButton } from "@canonical/react-components";
+import usePortal from "react-useportal";
+import CreateImageFromInstanceForm from "../forms/CreateImageFromInstanceForm";
+import { useInstanceLoading } from "context/instanceLoading";
+
+interface Props {
+  instance: LxdInstance;
+  isDeleting: boolean;
+}
+
+const CreateImageFromInstanceBtn: FC<Props> = ({ instance, isDeleting }) => {
```

Comment:
```suggestion
const CreateImageFromInstanceBtn: FC<Props> = ({ instance }) => {
```
Probably can keep the interface simple.